### ISSUE TITLE
No support for modulusLength 3072 bit

### ISSUE THREAD

**Issue by danhnn** [PeculiarVentures/webcrypto#41]
Currently, we want to use 3072-bit for RSA key's modulusLength but as I checked the current source code: https://github.com/PeculiarVentures/webcrypto/blob/a1760f21e2e190f4762e51bf9af97772cd02b134/src/mechs/rsa/rsa_es.ts#L51 

We only support 1024, 2048, and 4096 modulusLength. Could I ask that why we don't support 3072-bit value here?

**Comment by microshine** [comment]
@danhnn Thank you for pointing to this. 

### Chrome
![image](https://user-images.githubusercontent.com/7333669/143201405-c3c39b63-698f-4b1d-869d-c6f6e506f07c.png)

We should update the module

**Comment by microshine** [comment]
@danhnn I've fixed and published `@peculiar/webcrypto@1.2.3`.


```js
const alg = {
  name: "RSASSA-PKCS1-v1_5",
  hash: "SHA-256",
  publicExponent: new Uint8Array([1,0,1]),
  modulusLength: 3072,
};
const keys = await crypto.subtle.generateKey(alg, false, ["sign", "verify"]);
```

Please try it

**Comment by danhnn** [comment]
Thanks, @microshine!
